Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabComponentAdapter.cpp

```diff
@@ -264,4 +270,25 @@ namespace AzToolsFramework::Prefab
             return false;
         }
     }
+
+    bool PrefabComponentAdapter::IsComponentDisabled(const AZ::Component* component)
+    {
+        AZ_Assert(component, "Unable to check a component that is nullptr");
+
+        bool result = false;
+        AzToolsFramework::EditorDisabledCompositionRequestBus::EventResult(
+            result, component->GetEntityId(), &AzToolsFramework::EditorDisabledCompositionRequests::IsComponentDisabled, component);
```

Comment:
not sure you need to include the `AzToolsFramework::` aspects here because you're in the `AzToolsFramework` namespace.

Reply:
nice catch! will remove